ios/FluentUI/Resources/FluentUI-ios.xcassets/ic_fluent_dismiss_circle_24_regular.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_fluent_dismiss_circle_24_regular.pdf",
```

Comment:
isn't the latest guidiance svg (non scaled)?

Reply:
The guidance says there's not a significant difference for size or performance for pdf vs svg, and our other assets are pdf, so I matched them.